<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FragmentContextOpen</class>
 <widget class="QWidget" name="FragmentContextOpen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="FragmentTiles" name="fgTiles" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 66, 74);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FragmentPalettes" name="fgPalettes" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FragmentPreview" name="fgPreview" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="FragmentTilePreview" name="fgTilePreview" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 66, 74);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FragmentTilesets" name="fgTilesets" native="true"/>
     </item>
     <item>
      <widget class="FragmentTilesetProperties" name="fgTilesetProperties" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FragmentTiles</class>
   <extends>QWidget</extends>
   <header>fragmenttiles.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FragmentPreview</class>
   <extends>QWidget</extends>
   <header>fragmentpreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FragmentTilePreview</class>
   <extends>QWidget</extends>
   <header>fragmenttilepreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FragmentTilesets</class>
   <extends>QWidget</extends>
   <header>fragmenttilesets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FragmentTilesetProperties</class>
   <extends>QWidget</extends>
   <header>fragmenttilesetproperties.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FragmentPalettes</class>
   <extends>QWidget</extends>
   <header>fragmentpalettes.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
